Cargo.lock

```diff
@@ -1105,12 +1106,12 @@ dependencies = [
  "http",
  "httparse",
  "hyper 0.14.27",
- "hyper 1.0.0-rc.3",
+ "hyper 1.0.0-rc.4",
```

Comment:
The error is related to this dependency. So I'll try downgrading this back to `rc.3` in a bit